_i18n/en/resources/developer-guides/wallet-rpc.md

```diff
@@ -1,3 +1,4 @@
+{% include untranslated.html %}
```

Comment:
The English version doesn't need the 'untranslated' snippet

Reply:
I missed it. Sorry @erciccione

Reply:
I'll fix it, thanks for the catch @erciccione.